specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/examples/compute/ForceDeleteVirtualMachine.json

```diff
@@ -3,7 +3,7 @@
     "subscriptionId": "{subscription-id}",
     "resourceGroupName": "myResourceGroup",
     "vmName": "myVM",
-    "forceDeletion": "true",
+    "forceDeletion": true,
```

Comment:
why are we updating the forceDeletion value.

Reply:
This showed as error when i ran the example validation. the forcedeletion property is defined as boolean but in the example it is defined as string. So updated this to boolean to fix that issue.